addons/website/static/src/js/utils.js

```diff
@@ -6,106 +6,9 @@ var core = require('web.core');
 
 const { qweb, _t } = core;
 
-/**
- * Allows to load anchors from a page.
- *
- * @param {string} url
- * @returns {Deferred<string[]>}
- */
-function loadAnchors(url) {
```

Comment:
What about putting the refactoring part that moves these tool functions around in a distinct commit ?
This would make the other commit talk only about what is new.